CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
 # Welcome!
 
-If you are reading this document then you are interested in contributing to Islandora CLAW. All contributions are welcome: use-cases, documentation, code, patches, bug reports, feature requests, etc. You do not need to be a programmer to speak up!
+If you are reading this document then you are interested in contributing to Islandora 8. All contributions are welcome: use-cases, documentation, code, patches, bug reports, feature requests, etc. You do not need to be a programmer to speak up!
```

Comment:
If you're going to update CONTRIBUTING.md, can you throw these changes in there too? https://gist.github.com/manez/74b7fdda05cd01186c1d9b471c61b9dc 

I'm going to use that as the template across the board to get them all to match.